### PR TITLE
fix: env vars

### PR DESCRIPTION
**Pull request by Zamiell**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- Fixed some spelling/grammar.
- Added some links to documentation where appropriate.
- Comments in ".env.example" should not go past the 80 character line limit, which makes things easier to read for people with disabilities, people who work with two files side by side, etc.
- The env template should provide quotation marks, which prevents end-users from shooting themselves in the foot when they add a new environment variable with a special character.

Further questions:
- Should `SERVERVAR` be changed to `SERVER_VAR` and `NEXT_PUBLIC_CLIENTVAR` changed to `NEXT_PUBLIC_CLIENT_VAR`?

### CODE REVIEW

File: cli/src/installers/envVars.ts [hunk 1]
```diff
@@ -36,42 +36,51 @@ export const envVariablesInstaller: Installer = ({ projectDir, packages }) => {
...
 `;
 
   if (usingAuth)
     content += `
 # Next Auth
-# You can generate the secret via 'openssl rand -base64 32' on Linux
-# More info: https://next-auth.js.org/configuration/options#secret
-# NEXTAUTH_SECRET=
-NEXTAUTH_URL=http://localhost:3000
+# You can generate a new secret on Linux with: openssl rand -base64 32
```
Comment (juliusmarminge) [inline cli/src/installers/envVars.ts:56]: Unix

File: cli/src/installers/envVars.ts [hunk 2]
```diff
@@ -36,42 +36,51 @@ export const envVariablesInstaller: Installer = ({ projectDir, packages }) => {
...
+# https://next-auth.js.org/configuration/options#secret
+NEXTAUTH_SECRET=""
```
Comment (juliusmarminge) [inline cli/src/installers/envVars.ts:58]: This should be commented out by default since it will be handled automatically in dev

**Review by juliusmarminge — APPROVED** [review]

**Comment by Zamiell** [comment]
can you respond to this?

image

**Comment by juliusmarminge** [comment]
sorry - nah it's fine imo

**Pull request merged**